iiif_prezi3/helpers/canvas_helpers.py

```diff
@@ -23,5 +23,21 @@ def add_image(self, image_url, anno_id=None, anno_page_id=None, **kwargs):
         self.items.append(anno_page)
         return anno_page
 
+    def add_thumbnail(self, image_url, **kwargs):
+        """Adds a thumbnail to an existing canvas.
 
-monkeypatch_schema(Canvas, AddImageToCanvas)
+        Args:
+            image_url (str): An HTTP URL which points to the thumbnail.
+            **kwargs (): see ResourceItem.
+
+        Returns:
+            anno_page (AnnotationPage): the AnnotationPage with an Annotation and ResourceItem attached.
```

Comment:
I think this needs to be updated as the return for this helper is currently the canvas object.

It might be nice to return the ResourceItem instead in case further manipulation on it is desired (otherwise, the user should already have the canvas object in order to use this helper, so the return might be redundant).